Docs/architecture.md

```diff
@@ -15,8 +15,4 @@ However, customers looking for any of the following are expected to prefer a set
   - Gateway management.
 - Homogenous management of devices and gateways independent of connectivity technology.
```

Comment:
"," is missing before the word Independent  ; Homogenous management of devices and gateways, independent of connectivity technology.